app/main/menus.js

```diff
@@ -231,165 +36,148 @@ async function openFileCallback(mainWindow) {
     const filePath = filePaths[0];
     mainWindow.webContents.send('open-file', filePath);
   }
-}
+};
 
-async function saveAsCallback(mainWindow) {
+const saveAs = async (mainWindow) => {
```

Comment:
I usually prefer functions to anonymous as it is easier to read stacktraces when we have explicit names